Javascript Mutator not found

In my package.json: 
```js
    "stryker": "0.15.3",
    "stryker-api": "0.11.0",
    "stryker-baseline-reporter": "1.1.0",
    "stryker-html-reporter": "0.11.1",
    "stryker-javascript-mutator": "0.2.2",
    "stryker-mocha-framework": "0.7.0",
    "stryker-mocha-runner": "0.10.0",
```

When I run stryker I get the following deprecation warning: 
```
[WARN] ES5Mutator - DEPRECATED: The es5 mutator is deprecated and will be removed in the future. Please upgrade to the stryker-javascript-mutator (npm install --save-dev stryker-javascript-mutator) and set "mutator: 'javascript'" in your stryker.conf.js!
```

Which makes sense, but when I follow the instruction and add `mutator: 'javascript'` to my conf I get the error:

```
[ERROR] StrykerCli - an error occurred Error: Could not find a mutant-generator with name javascript, did you install it correctly (for example: npm install --save-dev stryker-javascript)?
    at MutatorFactory.Factory.create (/Users/iegner/Code/highstreet/node_modules/stryker-api/src/core/Factory.js:43:19)

```

--- a/packages/stryker/src/mutators/ES5Mutator.ts
+++ b/packages/stryker/src/mutators/ES5Mutator.ts
@@ -30,7 +30,7 @@ export default class ES5Mutator implements Mutator {
     new BooleanSubstitutionMutator()
   ]) {
     this.log = getLogger(ES5Mutator.name);
-    this.log.warn(`DEPRECATED: The es5 mutator is deprecated and will be removed in the future. Please upgrade to the stryker-javascript-mutator (npm install --save-dev stryker-javascript-mutator) and set "mutator: 'javascript'" in your stryker.conf.js!`);
+    this.log.warn(`DEPRECATED: The es5 mutator is deprecated and will be removed in the future. Please upgrade to the stryker-javascript-mutator (npm install --save-dev stryker-javascript-mutator) and set "mutator: 'javascript'" in your stryker.conf.js! If you have a plugins array in your stryker.conf.js, be sure to add 'stryker-javascript-mutator'.`);
   }
 
 

